mreg/tests.py

```diff
@@ -291,6 +299,13 @@ def test_model_can_create_ns(self):
         new_count = Network.objects.count()
         self.assertNotEqual(old_count, new_count)
 
+    def test_model_can_create_ipv6_ns(self):
+        """Test that the model is able to create an IPv6 Network."""
+        old_count = Network.objects.count()
+        clean_and_save(self.network_ipv6_sample)
+        new_count = Network.objects.count()
+        self.assertNotEqual(old_count, new_count)
```

Comment:
Just picked a random one here, but consider using `assertGreater/assertLess` instead of just `assertNotEqual`.